pkg/gofr/http/router.go

```diff
@@ -36,6 +47,109 @@ func (rou *Router) Add(method, pattern string, handler http.Handler) {
 	rou.Router.NewRoute().Methods(method).Path(pattern).Handler(h)
 }
 
+func (rou *Router) GetDefaultStaticFilesConfig() StaticFileConfig {
+	staticConfig := StaticFileConfig{
+		DirectoryListing: true,
+		HideDotFiles:     true,
+	}
+	return staticConfig
+}
+
+// Static File Handling.
+func (rou *Router) AddStaticFiles(endpoint string, staticConfig StaticFileConfig) {
+	fileServer := http.FileServer(http.Dir(staticConfig.FileDirectory))
+	rou.Router.NewRoute().PathPrefix(endpoint + "/").Handler(http.StripPrefix(endpoint, staticHandler(fileServer, staticConfig)))
+}
+
+// Check all the static handling configs.
+func staticHandler(fileServer http.Handler, staticConfig StaticFileConfig) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		url := r.URL.Path
+
+		const forbiddenBody string = "403 forbidden"
```

Comment:
This const should be moved out of the function to the package level, so you don't have to pass it to checkDotFiles,                         checkExcludedExtensions, and                         checkExcludedFiles

Reply:
moved the const to outside of function